package.json

```diff
@@ -14,6 +14,7 @@
     "WIP-build-client-webpack": "cross-env NODE_ENV=production SSR=true webpack --progress --config webpack.client.js",
     "prestart": "npm run build-less & npm run build-client-js & npm run build-server-js",
     "start": "cross-env SSR=true node ./lib/server/app.js",
+    "start-prod": "npm run build-client-js & npm run build-server-js & cross-env SSR=true node ./lib/server/app.js",
```

Comment:
Why this instead of `start`?
Keep in mind that `prestart` just above will be run first any time you call `start`.

I see you're running less at the end of the Dockerfile. What's the reason for doing it there instead of in npm scripts (build-less, run by default when running `npm start`) ?